tensorflow_lite_support/python/test/task/text/bert_clu_annotator_test.py

```diff
@@ -131,6 +133,73 @@ def test_annotate_model(self, model_name, model_file_type, clu_request,
     self.assertProtoEquals(text_clu_response.to_pb2(),
                            expected_clu_response.to_pb2())
 
+  @parameterized.parameters(
+    (_CLU_REQUEST, _CluResponse(
+      domains=[], intents=[], categorical_slots=[], noncategorical_slots=[
+      _NonCategoricalSlot(
+        slot='restaurant_name',
+        extraction=_Extraction(
+          value='Andes Cafe', score=0.914949, start=42, end=52))]),
+     0.99, None, 0.99, None),
+    (_CLU_REQUEST, _CluResponse(
+        domains=[], intents=[], categorical_slots=[], noncategorical_slots=[]),
+     0.99, None, 0.99, 0.99),
+  )
+  def test_thresholds(self, clu_request, expected_clu_response,
+                      domain_threshold, intent_threshold,
+                      categorical_slot_threshold, noncategorical_slot_threshold):
+    # Creates annotator.
+    base_options = _BaseOptions(file_name=self.model_path)
+    bert_clu_annotation_options = _BertCluAnnotationOptions(
+        domain_threshold=domain_threshold, intent_threshold=intent_threshold,
+        categorical_slot_threshold=categorical_slot_threshold,
+        noncategorical_slot_threshold=noncategorical_slot_threshold)
+    options = _BertCluAnnotatorOptions(
+        base_options=base_options,
+        bert_clu_annotation_options=bert_clu_annotation_options)
+    annotator = _BertCluAnnotator.create_from_options(options)
+
+    # Annotates CLU request using the given model.
+    text_clu_response = annotator.annotate(clu_request)
+    self.assertProtoEquals(text_clu_response.to_pb2(),
+                           expected_clu_response.to_pb2())
+
+  def test_max_history_turns(self):
+    # Creates annotator.
+    base_options = _BaseOptions(file_name=self.model_path)
+    bert_clu_annotation_options = _BertCluAnnotationOptions(
+        max_history_turns=10)
```

Comment:
The input only has 3 exchanges so setting max_history_turns=10 doesn't change the model output.

Could you either add more turns (sentences) to the conversation and adjust max_history_turns so that it doesn't cover the whole conversation? Ideally, I think you should change both _CLU_REQUEST and _CLU_RESPONSE, making the conversation longer, and here set a max_history_turns shorter than the conversation length to test that the output does change. Please note that the default value of max_history_turns is 5.